Automattic/wp-calypso
# Packages - Onboarding: add intent-screen

## Changes proposed in this Pull Request

Recently we created a new package `onboarding-components` and extracted the `IntentScreen` component into it. After doing this I noticed we already had a similar package called `onboarding`. To simplify and consolidate I thought it best to move `intent-screen` into `onboarding` and delete the newly created `onboarding-components` package.

This PR moves the IntentScreen component and all its children to `packages/onboarding` and removes `pacakges/onboarding-components`

#### Testing instructions

1. Pull branch and `yarn start`
2. Go to the intent-step of the signup - http://calypso.localhost:3000/start/
3. Check to make sure there are no errors and everything is functioning properly
4. Repeat the same with the stepper flow - http://calypso.localhost:3000/stepper


Related to #61240
